yarn.lock

```diff
@@ -6920,6 +6918,13 @@ debug@4.1.0:
   dependencies:
     ms "^2.1.1"
 
+debug@^4:
+  version "4.3.1"
+  resolved "https://registry.yarnpkg.com/debug/-/debug-4.3.1.tgz#f0d229c505e0c6d8c49ac553d1b13dc183f6b2ee"
+  integrity sha512-doEwdvm4PCeK4K3RQN2ZC2BYUBaxwLARCqZmMjtF8a51J2Rb0xpVloFRnCODwqjpwnAoao4pelN8l3RJdv3gRQ==
+  dependencies:
+    ms "2.1.2"
```

Comment:
We already depend on version 4 up on line 6907. Delete lines 6907-6912 and 6921-6926, then bootstrap again. Those dependencies should be merged after that.

Reply:
I see. thanks.